EasyPost/Insurance.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using RestSharp;
+
+namespace EasyPost
+{
+    public class Insurance : Resource
+    {
+        public decimal amount { get; set; }
```

Comment:
This is a string (we use string fields almost exclusively with a few exceptions).

Reply:
In the Java library, which I modeled this after, `amount` is a number, not a string: https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/Insurance.java#L22

Reply:
Unfortunately the API returns a string even if it was setup wrong in Java.

Reply:
Resolved in [f696919](https://github.com/EasyPost/easypost-csharp/pull/205/commits/f696919c16d57f48faeb4f74bedd73f684112b54)